### PR TITLE
build alert component

### PR DESCRIPTION
**Pull request by techieshubham30**

### CODE REVIEW

File: component/alert.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (HardikN19) [inline component/alert.html:8]: add css link here

**Review by HardikN19 — CHANGES_REQUESTED** [review]

File: component/alert.html (hunk 1 above)
Comment (techieshubham30) [reply component/alert.html:8]: thanks for the suggestion, will do that

**Review by HardikN19 — APPROVED** [review]
Done

**Review by HardikN19 — APPROVED** [review]
Done

**Comment by HardikN19** [comment]
Done with approval

**Pull request merged**